src/sparseml/export/helpers.py

```diff
@@ -11,29 +11,104 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 import os
 import shutil
 import tarfile
 from collections import OrderedDict
 from enum import Enum
 from pathlib import Path
-from typing import Callable, List, Union
+from typing import Callable, Dict, List, Optional, Union
 
 import onnx
 
+from sparseml.exporters import ExportTargets
 from sparsezoo.utils.onnx import save_onnx
 
 
-__all__ = ["apply_optimizations", "export_sample_inputs_outputs"]
+__all__ = [
+    "apply_optimizations",
+    "export_sample_inputs_outputs",
+    "create_deployment_folder",
+    "AVAILABLE_DEPLOYMENT_TARGETS",
+    "ONNX_MODEL_NAME",
+]
 
+AVAILABLE_DEPLOYMENT_TARGETS = [target.value for target in ExportTargets]
+ONNX_MODEL_NAME = "model.onnx"
+ONNX_DATA_NAME = "model.data"
+DEFAULT_DEPLOYMENT_DIRECTORY_STRUCTURE = {ONNX_MODEL_NAME: {}}
 
-class GraphOptimizationOptions(Enum):
+
+def create_deployment_folder(
+    source_path: Union[Path, str],
+    target_path: Union[Path, str],
+    deployment_directory_name: str = "deployment",
+    deployment_directory_structure: Optional[Dict] = None,
+    onnx_model_name: Optional[str] = None,
+) -> str:
```

Comment:
should allow for a list of files to be copied in (will make it easier for other integrations to just call in with their list of configs, tokenizers, etc)

Reply:
I use here dictionary instead of list, but this may be an overkill. Let's discuss.